pest/src/inputs/position.rs

```diff
@@ -4,48 +4,44 @@
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
+#[allow(unused_imports)] use std::ascii::AsciiExt;
```

Comment:
I would remove the attribute until the next version of Rust is stable.

Reply:
:+1: